__tests__/balance.test.js

```diff
@@ -0,0 +1,32 @@
+import TestUtils from "./test-utils";
+
+describe("balance api", () => {
+  it("should return 200 with a valid body", async () => {
+    const response = await TestUtils.request
+      .get("/wallet/balance")
+      .set({ "x-wallet-id": TestUtils.walletId });
+    expect(response.status).toBe(200);
+    expect(response.body.available).toBe(83199);
+    expect(response.body.locked).toBe(0);
```

Comment:
Maybe we should add a locked utxo in the fixtures to test this. All balances have locked as 0

Reply:
Done